src/common.rs

```diff
@@ -3822,9 +3822,19 @@ impl Components {
     /// components.refresh();
     /// ```
     pub fn refresh(&mut self) {
-        for component in self.list_mut() {
-            component.refresh();
+        #[cfg(feature = "multithread")]
```

Comment:
Instead of doing it like this, could you instead create an `iter_mut()` function into `utils.rs` (which would be the same as `into_iter` except it would call `par_iter_mut`/`iter_mut`) please? The idea is to prevent the duplication of `.for_each(|component| component.refresh())`. It might seem super small but it's easy to lose track of such things quite easily.